huaweicloud/services/rms/resource_huaweicloud_rms_originational_assignment_package.go

```diff
@@ -0,0 +1,404 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product Config
+// ---------------------------------------------------------------
+
+package rms
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// API: Config POST /v1/resource-manager/organizations/{organization_id}/conformance-packs
+// API: Config GET /v1/resource-manager/organizations/{organization_id}/conformance-packs/{conformance_pack_id}
+// API: Config DELETE /v1/resource-manager/organizations/{organization_id}/conformance-packs/{conformance_pack_id}
+// API: Config GET /v1/resource-manager/organizations/{organization_id}/conformance-packs/statuses
+func ResourceOrgAssignmentPackage() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceOrgAssignmentPackageCreate,
+		ReadContext:   resourceOrgAssignmentPackageRead,
+		DeleteContext: resourceOrgAssignmentPackageDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceOrgAssignmentPackageImportState,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
```

Comment:
region is not necessary for global service

Reply:
done